### ISSUE TITLE
Overwrite folder

### ISSUE THREAD

**Issue by Schizosaccaromyces** [TanninOne/modorganizer#695]
It seems that there are issues with the Overwrite folder in MO. When the settings are left as default, it works, but as soon as they are changed it seems that MO does not read any files in the structure; however, other tools such as XEdit can open and read those files when launched through MO, but when they save they throw a "cannot rename" error and generate a file with a time and date in the filetype (same format as the backups)

**Comment by Maxunit** [comment]
Yep, this happens on my end as well. XEdit cannot save backup files to the destination folder, but instead keeps them in the overwrite directory.

**Comment by mgerrior68** [comment]
Both are happening with XEdit for me. I cannot change location of overwrite and even when i leave at default, any edited files are dumped to overwrite and not back to original mod folder.
